### ISSUE TITLE
[Feature Request] - Standalone App

### ISSUE THREAD

**Issue by SyndiShanX** [FTBTeam/FTB-App#835]
### What is your suggestion?

It would be nice to have the FTB App disconnected from Overwolf, having Overwolf installed just for this app is kinda annoying an it would be nice if there was a Standalone App (Even a barebones alternative)

**Comment by MichaelHillcox** [comment]
As of right now, there is no major movement on this topic since last time it was suggested. Although we see there is some user request for this, the majority of people don't seem to mind too much. 

We'll be sure to update this issue if the situation changes.

**Comment by Highlanderwolf** [comment]
I'd hazard a guess that the majority of people that would like to see the standalone FTB client abandoned the idea when Twitch and Curse took over and mothballed the original client. I looked to see if there was one on a whim, having been using the AT Launcher for some time now. Personally I liked the FTB launcher better than AT, but I'm not installing Twitch, or Overwolf, to use it.

**Comment by Wyvernity** [comment]
Just wanna play modded easily and don't wanna download other unnecessary stuff. If you wanna put ads, that's not a problem. Please just do it on your own app.

**Comment by MichaelHillcox** [comment]
Closing as it's not planned but again, we'll update this if this stance changes

**Comment by Stapleton** [comment]
I agree with this and many other people I know just straight up don't want extra software coming with the desired software. Yeah dependencies are a common thing, but those are not another monolithic program with their own dependencies, startup routines, app data, etc. 

Personally my gripes are having to neuter an extra application from starting with my system and potentially causing issues just to play a FTB branded pack. If you could still get the Overwolf benefits without the entirety of Overwolf itself, causing this to be standalone where it matters to me and what I would guess to be many other people with minimal desire to share their opinions.

Maybe a poll on the discord would garner more support for this.

**Comment by MichaelHillcox** [comment]
We're in the process of actually making this a reality. We're just waiting on an EV Cert for our standalone windows application. I'll keep you all posted. Once this is sorted, you will no longer need Overwolf Installed to use the FTB app. It's worth noting that our ads will still be powered by Overwolf but you won't need their platform to use our app.
